cmd/beach/cmd/resource-download.go

```diff
@@ -98,15 +98,15 @@ func handleResourceDownloadRun(cmd *cobra.Command, args []string) {
 			log.Error(err)
 		} else {
 			source := bucket.Object(attributes.Name)
-			targetPath := filepath.Dir(sandbox.ProjectDataPersistentResourcesPath + "/" + getRelativePersistentResourcePathByHash(attributes.Name) + attributes.Name)
+			targetPath := filepath.Dir(sandbox.ProjectDataPersistentResourcesPath + "/" + getRelativePersistentResourcePathByHash(attributes.Name) + "/")
 
 			err = os.MkdirAll(targetPath, 0755)
 			if err != nil {
 				log.Fatal(err)
 				return
 			}
 
-			file, err := os.OpenFile(targetPath+"/"+filepath.Base(attributes.Name), os.O_RDWR|os.O_CREATE, 0644)
+			file, err := os.OpenFile(targetPath+"/"+filepath.Dir(attributes.Name), os.O_RDWR|os.O_CREATE, 0644)
```

Comment:
this works reliably? I guess as long as we don't change how we store them in the cloud storage...

Reply:
… and I can't think of a reason for changing it. 😎